Building.md

```diff
@@ -123,27 +123,27 @@ those come from inside the wasm build. You can strip out these symbols and other
 info, that won't be usable when running anyway. For this we use `wasm-opt` from the
 [enscripten toolchain](). This is a bunch of C++ code that needs to be compiled, and to
 simplify the whole process, as well as create reproduceable builds, we have created
-[`cosmwasm-opt`](https://github.com/confio/cosmwasm-opt), 
+[`cosmwasm-opt`](https://github.com/confio/cosmwasm-opt),
 which contains a `Dockerfile` that you can use to run both `wasm-pack` and `wasm-opt`.
 To make the build, just run the following in the project directory:
 
-```shell script
+```sh
 docker run --rm -u $(id -u):$(id -g) -v $(pwd):/code confio/cosmwasm-opt:0.4.1
 
 du -h contract.wasm
 ```
 
 Note that this always outputs the file as `contract.wasm`, not with the name of
 the project (yes, every tool chain has a different output location).
-For the hackatom sample, I now get down to 52kB, an 18% improvement. 
+For the hackatom sample, I now get down to 52kB, an 18% improvement.
 This is as far as you can minimize the input, without removing actual functionality.
 
 While we cannot trim down the wasm code anymore, we can still reduce the size a bit
 for loading in blockchain transactions. We [soon plan](https://github.com/confio/go-cosmwasm/issues/20)
 to allow gzip-ed wasm in the transactions posted to the chain, which will further
 reduce gas cost of the code upload. Check out this final output:
 
-```shell script
+```sh
```

Comment:
I think this is smaller  now

Reply:
The whole document could use an overhaul. But I would avoid doing that as part of this big PR.